airbyte-integrations/connectors/source-pinterest/source_pinterest/spec.json

```diff
@@ -0,0 +1,50 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/pinterest",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Pinterest Spec",
+    "type": "object",
+    "required": ["client_id", "client_secret", "refresh_token", "start_date"],
```

Comment:
start_date should not be required. If the user doesn't input a start date then we should use some old date by default e.g: 2010.

Reply:
Add validation for start_date. Default value is 2020-07-28.